### ISSUE TITLE
Math not rendered properly?

### ISSUE THREAD

**Issue by apuignav** [lhcb/starterkit-lessons#9]
In https://lhcb.github.io/starterkit-lessons/second-analysis-steps/building-decays.html, I don't see any math, and actually the callout box is cut.

See screenshot below:
<img width="1552" alt="screen shot 2017-06-20 at 17 13 15" src="https://user-images.githubusercontent.com/688368/27340824-c5e4bea4-55db-11e7-8a36-68ac5b9d869f.png">

Is it only me?

**Comment by henryiii** [comment]
You can't put any non-standard markdown in GitBook blocks with the current implementations I've seen. This is causing problems for my terminal additions for DevelopKit, as well. Even the official styled blocks (gitbook-plugings-hints) do not support non-standard markdown inside them.

There needs to be a way to reenter the standard processing procedure inside a block.

**Comment by alexpearce** [comment]
Indeed, this is annoying, and is why we have a few custom GitBook plugins. I was hoping to get these problems fixed before making an 'official' announcement with the new URL for the lessons (and redirecting people from the old sites).
